README.md

```diff
@@ -1,7 +1,31 @@
+# Figma Plugin API typings
+[![npm](https://img.shields.io/npm/v/@figma/plugin-typings?logo=npm&cacheSeconds=1800)](https://www.npmjs.com/package/@figma/plugin-typings)
+
 This repository contains the typings for the Figma Plugin API.
 
-It is intended to be installed as "@figma/plugin-typings" and the types should become globally available without needing to use import statements. We do it this way because the plugin API is part of the host environment, as opposed to being a package that a plugin includes.
+## Usage
+
+1. Installation
+    ```sh
+    npm i --save-dev @figma/plugin-typings
+    # or
+    yarn add -D @figma/plugin-typings
+    ```
+
+2. Configure _tsconfig.json_
+    ```js
+    {
+        "compilerOptions": {
+            "typeRoots": [
+                "./node_modules/@types",
+                "./node_modules/@figma"
+            ]
+        }
+    }
+    ```
+    With config above, TypeScript will parse provided list of `"typeRoots"` folders for global available types (by default it's `"./node_modules/@types"`). Types should become globally available without needing to use import statements. We do it this way because the plugin API is part of the host environment, as opposed to being a package that a plugin includes.
```

Comment:
How about this:

> The configuration above is needed for the TypeScript compiler to use type definitions found in both `./node_modules/@types` and `./node_modules/@figma`. Normally, most type definitions are from DefinitelyTyped and are installed in `/@types`, which is then automatically included by TypeScript. Since we host the plugin typings separately, they are installed outside in `/@figma` instead.

The part about the typings being globally available is an entirely separate point. Putting it in the same paragraph is confusing, so I would leave it at the top.

Reply:
> leave it at the top

putted it to separate paragraph